docs/user_guides/single_table/copulagan.rst

```diff
@@ -343,6 +343,36 @@ Now that we have discovered the basics, let's go over a few more
 advanced usage examples and see the different arguments that we can pass
 to our ``CopulaGAN`` Model in order to customize it to our needs.
 
+Setting Bounds for Numerical Columns
```

Comment:
What do you think about adding this section to the `CTGAN` and `TVAE` user guides too @amontanez24 ?